things/api.py

```diff
@@ -90,9 +90,10 @@ def tasks(uuid=None, include_items=False, **kwargs):  # noqa: C901
         If the argument is `True`, only include tasks _with_ a due date.
         If the argument is `None` (default), then include all tasks.
 
-    search_query : str, bool, or None, optional
+    search_query : str, optional
```

Comment:
I think it's not `optional`

Reply:
It is optional. Remember, we are in the `tasks` function here.